src/routes/safe/components/Transactions/TxList/TxSummary.tsx

```diff
@@ -51,31 +63,55 @@ export const TxSummary = ({ txDetails }: Props): ReactElement => {
           <TxShareButton safeTxHash={txDetails.detailedExecutionInfo.safeTxHash} />
         </div>
       )}
+      {txData?.operation === Operation.DELEGATE && (
+        <div className="tx-operation">
+          <DelegateCallWarning isKnown={isCustomTxInfo(txInfo) && !!txInfo?.to?.name} />
+        </div>
+      )}
+      {isMultiSendTxInfo(txInfo) && (
+        <>
+          <TxInfoMultiSend txInfo={txInfo} />
+          <br />
```

Comment:
Is this not possible with CSS?

Reply:
We are using <br /> in more places and here it does the job unless we want to go pixel perfect with the designs atm.